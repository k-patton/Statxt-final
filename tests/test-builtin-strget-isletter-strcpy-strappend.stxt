int main() {
	string str1;
	str1 = "hel*o#";
	char c1;
	char c2;
	c1 = strget(str1, 1);
	c2 = strget(str1, 3);
	printchar(c1);
	printchar(c2);
	
	bool b1;
	bool b2;
	b1 = isletter(c1);
	b2 = isletter(c2);
	printb(b1);
	printb(b2);

	string str2;
	str2 = calloc(strlen(str1), 1);
	str2 = strcpy(str2, str1);
	strappend(str2,3,'l');
	printstr(str2);
}