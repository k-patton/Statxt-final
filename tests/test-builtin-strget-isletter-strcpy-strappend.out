e
*
1
0
hello#
